features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2884,6 +2884,10 @@
             {% endfor %}
         </ChoreoDomains>
         {% endif %}
+
+        <AuthenticatorNameInAuthConfig>
```

Comment:
better to add a small comment explaining the purpose of this config. 
also let's get this config added to identity.xml

Reply:
Fixed